app/backend/src/clj/teet/integration/x_road.clj

```diff
@@ -155,16 +155,73 @@
               :omandiosa_suurus (omandi-get ox :a:omandiosa_suurus)})
            owner-xml-seq)}))
 
+(defn d-jagu34* [key children sequential-vals?]
```

Comment:
this fn body could use a docstring and some comments... pretty terse and not obvious what it does